### PR TITLE
feat: alert refactoring

### PR DESCRIPTION
**Pull request by InnaAtanasova**
- adds theme designer variables for colors 
- updates paddings and spacing according to Fiori3 guidelines
- removes extra code

### CODE REVIEW

File: scss/components/alert.scss [hunk 1]
```diff
@@ -1,45 +1,66 @@
...
 @import "./../mixins";
-@import "./../functions";
 @import "./../icons/mixins";
-/*!
-.fd-alert+(--information, --success, --warning, --error, --no-icon)
-    .fd-alert__close
-*/
+
+@import "./../../theme-designer/Base/baseLib/sap_fiori_3/sass_variables.scss";
+@import "./../../theme-designer/Base/baseLib/sap_fiori_3/css_variables";
```
Comment (jbadan) [inline scss/components/alert.scss]: Should wait for https://github.com/SAP/fundamental-styles/pull/239 to be merge and then import from `@sap-theming/theming-base-content`?

Comment (droshev) [reply scss/components/alert.scss]: this is done :)

File: components/alert.scss [hunk 2]
```diff
@@ -1,6 +1,9 @@
 $fd-icons-path: "../scss/icons/";
 $fd-fonts-path: "../scss/fonts/";
 
+@import "./../node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/sass_variables.scss";
+@import "./../node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/css_variables";
+
```
Comment (jbadan) [inline components/alert.scss]: Do we need the `scss` reference here if it's already defined in settings that are being imported into alert.scss?

Comment (InnaAtanasova) [reply components/alert.scss]: We don't need it, I removed it. Thanks for noticing it

Comment (InnaAtanasova) [reply components/alert.scss]: We don't need it, I removed it. Thanks for noticing it

**Comment by jbadan** [comment]
@InnaAtanasova can you merge with master to bring in those latest changes for easier review (I know, easier said than done 😆 ). Thank you!

**Pull request closed without merging**